platform/marvell-arm64/platform.conf

```diff
@@ -2,27 +2,66 @@
 
 # over ride default behaviour
 
+PLATFORM_AC5X="arm64-nokia_ixs7215_52xb-r0"
+
 echo "Preparing for installation ... "
 
 IS_MASS=0
 # global defines
 kernel_addr=0x1100000
-fdt_addr=0x1000000
-fit_addr=0x1000000
-initrd_addr=0x2000000
+
+if [ "$install_env" = "onie" ]; then
+    MACH_FILE="/etc/machine.conf"
+else
+    MACH_FILE="/host/machine.conf"
+fi
+
+PLATFORM=`sed -n 's/^onie_platform=\(.*\)/\1/p' $MACH_FILE`
+echo "Intalling SONiC from $install_env on Platform $PLATFORM"
+
+if [ "$PLATFORM" = "$PLATFORM_AC5X" ]; then
+    fdt_addr=0x201000000
+    fit_addr=0x210000000
+    initrd_addr=0x206000000
+else
+    fdt_addr=0x1000000
+    fit_addr=0x8000000
+    initrd_addr=0x2000000
+fi
+
 VAR_LOG=512
-DISK_SIZE=9
+DISK_SIZE=14
+
+kernel_fname="/boot/vmlinuz-5.10.0-18-2-arm64"
+initrd_fname="/boot/initrd.img-5.10.0-18-2-arm64"
+
+if [ "$PLATFORM" = "$PLATFORM_AC5X" ]; then
+    fdt_fname="/boot/ac5x.dtb"
+else
+    fdt_fname="/boot/armada-7020-comexpress.dtb"
```

Comment:
i think it is better to put these platform specific variable assignment in one platform. block 22-30 and this block should be merged, there are other example below should be merged as well.